<launch>
  <include file="$(find dualcamerasaver)/launch/defs.machine"/>

  <!-- The storer has to be called on the machine that has all the saved files!/-->
    <node name="imagestorer" machine="gigabyte" pkg="dualcamerasaver" type="imagestorer" output="screen">

      <param name="save_dir"    value="/tmp"/>
      <param name="num_images"  value="6"/>
      <param name="store_dir"    value="/tmp/store"/>
      <param name="loop_rate"   value="1"/>
      <!--param name="sec_per_frame" value="0.5"/
      SO, because of the way I am saving stuff, I can't really save more than 1
      set of images per second, so maximum value of the parameter loop_rate is 1.
      -->
    </node>
</launch>
